support-frontend/app/views/windowGuardianPaymentConfig.scala.html

```diff
@@ -58,6 +58,17 @@
     REGULAR: window.guardian.stripeKeyDefaultCurrencies.REGULAR
   };
 
+  window.guardian.stripeKeyTortoiseMedia = {
+    ONE_OFF: {
+      default: "@paymentMethodConfigs.oneOffDefaultStripeConfig.tortoiseMediaAccount.rawPublicKey",
+      test: "@paymentMethodConfigs.oneOffTestStripeConfig.tortoiseMediaAccount.rawPublicKey"
```

Comment:
Since these won't (/shouldn't!) ever be used, should we set them to something unusable?

Reply:
Oh good question, I guess they both achieve the same thing. Hmm I'm not sure which one I prefer.